src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj

```diff
@@ -29,6 +29,11 @@
     <ProjectSystemLayer>VisualStudio</ProjectSystemLayer>
     <ProducingSignedVsix>true</ProducingSignedVsix>
     <IsProductComponent>true</IsProductComponent>
+    <InstallRoot>Default</InstallRoot>
+    <Ngen>true</Ngen>
+    <NgenApplication>vsn.exe</NgenApplication>
+    <NgenArchitecture>All</NgenArchitecture>
```

Comment:
All covers both 32-bit and 64-bit. Which also ensures that any OOP work or future work is covered.